exercises/risk-risiko.cpp

```diff
@@ -27,3 +27,92 @@
   M 3 vs 3 => blue win
   O 2 vs 1 => red win
 */
+
```

Comment:
hai caricato un file binario, male! mettilo in `.gitignore`

Reply:
Devo decisamente levare il vizio del "git add ." Risolvo immediatamente, grazie!